src/Http/Controllers/AddressController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+declare(strict_types=1);
+
+namespace Tychovbh\Mvc\Http\Controllers;
+
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+use Tychovbh\Mvc\Http\Controllers\AbstractController;
+
+class AddressController extends AbstractController
+{
+    /**
+     * @param Request $request
+     * @return JsonResource
+     */
+    public function store(Request $request): JsonResource
```

Comment:
House number and zipcode should be mandatory create a FormRequest class that validates the request, see StoreInvite in src/http/requests

(also see comment in AddressRoute you need to add something to make your validation work)